source/braille.py

```diff
@@ -1677,11 +1679,13 @@ def previousLine(self, start=False):
 					pass
 				else:
 					dest = dest.obj.makeTextInfo(textInfos.POSITION_LAST)
-					dest.expand(unit)
-			else:  # no page turn support
+			else:
+				# no page turn supportcode ..\nvda
```

Comment:
It looks like some mess ended up in here
```suggestion
				# no page turn support
```